atomic_reactor/schemas/content_manifest.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Image Content Manifest v1 definition",
+
+  "type": ["object", "null"],
+  "properties": {
+    "metadata": {
+      "description": "Information about the Image Content Manifest (ICM) file",
+      "type": "object",
+      "properties": {
+        "icm_version": {
+          "description": "ICM Version",
+          "type": "integer",
+          "default": 1
+        },
+        "icm_spec": {
+          "description": "URL for the ICM specification (this document)",
+          "type": "string",
+          "example": "https://github.com/containerbuildsystem/atomic-reactor/blob/COMMIT_REF/atomic_reactor/schemas/content_manifest.json"
```

Comment:
Latest JSON Schema draft defines ["examples"](https://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.9.5) as the keyword for examples (and it must be an array), but since we are using draft 4 here (which does not define any example keyword), it is probably fine.